disputer/test/Disputer.js

```diff
@@ -56,17 +56,17 @@ contract("Disputer.js", function(accounts) {
     await collateralToken.mint(sponsor3, toWei("100000"), { from: contractCreator });
     await collateralToken.mint(liquidator, toWei("100000"), { from: contractCreator });
     await collateralToken.mint(disputeBot, toWei("100000"), { from: contractCreator });
+  });
```

Comment:
moved the creation of the `mockOracle` to the `beforeEach` from the `before` to decouple the tests.